src/mtype.d

```diff
@@ -3492,7 +3492,7 @@ public:
             case Tcomplex80:
             case Timaginary80:
             case Tfloat80:
-                fvalue = Port.ldbl_max;
+                fvalue = real.max;
```

Comment:
This is wrong.  There is (unfortunately) no guarantee that real is 80-bit, or that host and target platform are the same.